amf-aml/shared/src/test/scala/amf/dialects/DialectInstancesCustomValidationTest.scala

```diff
@@ -21,55 +20,4 @@ class DialectInstancesCustomValidationTest extends DialectInstanceValidation wit
       assertReport(_, golden.map(g => s"$path/$g"))
     }
   }
-
-  def validateWithCustomProfile(dialect: String,
```

Comment:
Should we erase tests???

Reply:
Si nadie puede usar el feature, nisiquiera nosotros mismos, podriamos borrarlo (de hecho no habria manera de hacer le test).
Tambien diria de borrar el validation profile de AMF, ya que es una version vieja y generaria confunsion .